docs/UserGuide.md

```diff
@@ -69,8 +69,9 @@ Please take note of these few terms. It is explained as below:
 | Apple MacOS       | [Terminal](https://support.apple.com/en-sg/guide/terminal/apd5265185d-f365-44cb-8b09-71a064a42125/mac)     |
 | (Ubuntu) Linux    | [Bash Terminal](https://ubuntu.com/tutorials/command-line-for-beginners#3-opening-a-terminal)              |
 
-> For Microsoft Windows operating systems, the Command Prompt (CMD) is **not** recommended for use with WerkIt!
-> as some texts in the application will not be displayed correctly.
+> For Microsoft Windows operating systems, if you are having display issues (e.g. colours not displaying) with
+> your terminal, please see "My terminal in Windows is not displaying any colours in the application" 
+> in the [FAQ section](#frequently-asked-questions-faq).
```

Comment:
Nicely done but perhaps a consideration is we might want to number or FAQ so its easier for users to find a question based on its question number instead of finding based on title. In the long run, if we have a longer FAQ, finding this question without the question number might be difficult?

Reply:
Thank you for your feedback. This change has been implemented in Commit 5a92004